src/energyplus/ForwardTranslator/ForwardTranslateLoadProfilePlant.cpp

```diff
@@ -58,6 +58,21 @@ namespace energyplus {
       }
     }
 
+    // PlantLoopFluidType
+    if ((s = modelObject.plantLoopFluidType())) {
+      idfObject.setString(LoadProfile_PlantFields::PlantLoopFluidType, s.get());
+    }
+
+    // DegreeofSubCooling
+    if ((value = modelObject.degreeofSubCooling())) {
+      idfObject.setDouble(LoadProfile_PlantFields::DegreeofSubCooling, value.get());
+    }
+
+    // DegreeofLoopSubCooling
+    if ((value = modelObject.degreeofLoopSubCooling())) {
+      idfObject.setDouble(LoadProfile_PlantFields::DegreeofLoopSubCooling, value.get());
+    }
```

Comment:
Not optional, doesn't make sense to go through an optional indirection, just set directly.

```suggestion
    // PlantLoopFluidType
    idfObject.setString(LoadProfile_PlantFields::PlantLoopFluidType, modelObject.plantLoopFluidType());

    // DegreeofSubCooling
    idfObject.setDouble(LoadProfile_PlantFields::DegreeofSubCooling, modelObject.degreeofSubCooling());

    // DegreeofLoopSubCooling
    idfObject.setDouble(LoadProfile_PlantFields::DegreeofLoopSubCooling, modelObject.degreeofLoopSubCooling());
```